samoa-api/src/main/java/org/apache/samoa/streams/FileStream.java

```diff
@@ -52,9 +49,18 @@
       's', "Source Type (HDFS, local FS)", FileStreamSource.class,
       "LocalFileStreamSource");
 
+  public IntOption classIndexOption = new IntOption("classIndex", 'c',
+          "Class index of data. 0 for none or -1 for last attribute in file.", -1, -1, Integer.MAX_VALUE);
+
+  private FloatOption floatOption = new FloatOption("classWeight", 'w', "", 1.0);
```

Comment:
Would instance weights work in this case?
Could you summarize the difference between class weights and instance weights?